docs/Options/APIReference.rst

```diff
@@ -1,82 +1,101 @@
-# REST API
 
-> Author: [bibekdahal](https://github.com/bibekdahal)
+📚 API Reference
++++++++++++++++++
+
+
+Deep uses both GraphQL and REST.
+Most of the platform uses GraphQL but there are some parts where we still use 
+REST which we are trying to migrate from as well.
+
+REST Endpoints
+------------------
+
+Rest API Endpoints can be found here  : https://prod-api.thedeep.io/api-docs/
+
+
+GraphQL Endpoints
+--------------------
+
+GraphQL Endpoints points can be found here  : https://prod-api.thedeep.io/graphql-docs/
+
+REST API
+----------
 
 A thorough documentation of the API itself can be found at */api-docs/*.
 
-## Authentication
+**Authentication**
 
 Authentication is done using JSON Web Tokens passed in through the HTTP authorization header.
 
 Format:
 
-```
-Authorization: Bearer <JWT>
-```
+.. code-block:: bash  
+
+ Authorization: Bearer <JWT>
```

Comment:
We don't use JWT-based authentication anymore.

For the core deep, client -> backend, we use session-based authentication instead.
Most of the external clients use basic auth for now.

Let's remove this, and maybe we can look into a token-based authentication setup for external users if needed. But this will require changes in both backend + the frontend. cc: @AdityaKhatri @nayid-DFS

Reply:
Let's remove all the references of JWT